database/factories/PostFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+class PostFactory extends Factory
+{
+
+    /**
+     * The name of the post's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = \App\Models\Post::class;
```

Comment:
use cái model này lên trên nhé em. để sau này có sửa đổi thì sẽ dễ hơn
```suggestion
    protected $model = Post::class;
```